rotkehlchen/exchanges/bitmex.py

```diff
@@ -307,12 +351,19 @@ def query_online_history_events(
             start_ts: Timestamp,
             end_ts: Timestamp,
     ) -> Sequence['HistoryBaseEntry']:
-        resp = self._api_query_list('get', 'user/walletHistory', {'currency': 'all'})
+        self.first_connection()
+        resp = self._api_query('user/walletHistory', {'currency': 'all'})
 
         log.debug('Bitmex deposit/withdrawals query', results_num=len(resp))
         movements = []
         for movement in resp:
             try:
+                timestamp = iso8601ts_to_timestamp(movement['timestamp'])
+                if timestamp < start_ts:
+                    continue
+                if timestamp > end_ts:
+                    continue
```

Comment:
Can't this be combined?

```suggestion
                if end_ts < timestamp < start_ts:
                    continue
```